### ISSUE TITLE
[MU4 Issue] Audio output open fail - Linux (Arch) 

### ISSUE THREAD

**Issue by luciorgomes** [musescore/MuseScore#14167]
**Describe the bug**
No output is enabled (in beta MuseScore-4.0.0.3289917644-x86_64.AppImage) . 

**To Reproduce**
It happens on the program load. 
In Preferences > I/O (E/S in Portuguese) there is no option for Linux outputs (Alsa/Jack/Pulseaudio), only "System default" with strange values to buffer size. 

**Expected behavior**
A connection to the OS outputs when the program starts or some kind of selection options in Preferences

**Screenshots**
![Captura de tela de 2022-10-29 11-58-57](https://user-images.githubusercontent.com/25042989/198838977-520ce5ca-5beb-432c-ad21-7534ee05719a.png)
![Captura de tela de 2022-10-29 12-02-24](https://user-images.githubusercontent.com/25042989/198839003-29b9b4dd-4016-48f6-980b-0b49eeb79358.png)

**Platform information**
 - OS: Linux

**Additional context**
Terminal output:
> '/run/media/luciorg/HD Externo/home/bin/MuseScore-4.0.0.3289917644-x86_64.AppImage'
/usr/lib/libjack.so.0
/usr/lib/libnss3.so
QEventLoop: Cannot be used without QApplication
QEventLoop: Cannot be used without QApplication
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
11:50:53.940 | ERROR | main_thread | MuseSamplerLibHandler | MuseSamplerLibHandler: Unable to open MuseSampler library, path: libMuseSamplerCoreLib.so
11:50:53.940 | ERROR | main_thread | MuseSamplerResolver | MuseSamplerResolver: Incompatible MuseSampler library; ignoring
11:50:53.960 | INFO  | main_thread | GlobalModule | onPreInit: log path: /home/luciorg/.local/share/MuseScore/MuseScore4/logs/MuseScore_221029_115053.log
11:50:53.960 | INFO  | main_thread | GlobalModule | onPreInit: === Started MuseScore 4.0.0, build number 3289917644 ===
11:50:54.104 | INFO  | main_thread | DiagnosticsModule | onInit: success start crash handler
11:50:54.108 | WARN  | main_thread | Qt         | QIODevice::read (QFile, "/home/luciorg/.local/share/MuseScore/MuseScore4/shortcuts.xml"): device not open
11:50:54.109 | WARN  | main_thread | Qt         | QIODevice::read (QFile, "/home/luciorg/.local/share/MuseScore/MuseScore4/midi_mappings.xml"): device not open
11:50:54.135 | ERROR | main_thread | AudioModule | onInit: audio output open failed
11:50:54.424 | WARN  | main_thread | CloudService | readTokens: Could not find the tokens file: /home/luciorg/.local/share/MuseScore/MuseScore4/cred.dat
11:50:54.435 | WARN  | main_thread | IpcSocket  | connect: failed connect to server
11:50:54.435 | WARN  | main_thread | IpcSocket  | connect: failed connect to server
11:50:54.436 | INFO  | main_thread | IpcSocket  | connect: success connected to ipc server
11:50:54.436 | INFO  | 140587894302400 | IpcServer  | listen: id: "3fd36e70eca0482db15c79a8c4ef35dd"
11:51:11.461 | WARN  | main_thread | Qt         | QQmlComponent: Created graphical object was not placed in the graphics scene.
11:51:12.853 | INFO  | main_thread | ActionsDispatcher | dispatch: try call action: preference-dialog

**Comment by lyrra** [comment]
Musescore can't find the alsa device:
_11:50:54.135 | ERROR | main_thread | AudioModule | onInit: audio output open failed_

Could you try verify that your default alsa device works by playing some WAV file:
``` aplay -D default some_wav_file.wav ```

Also list your devices, and ensure that 'default' is listed:
``` 
aplay -L
...
default
hw:CARD=HDMI,DEV=3
    HDA ATI HDMI, HDMI 0
    Direct hardware device without any conversions
...
```

**Comment by Tantacrul** [comment]
So are you getting no sound when you enter notes and press play? Or is it accurately finding your default?

**Comment by luciorgomes** [comment]
It was an issue related to Pipewire (that I uninstalled to use Pulseaudio and Jack). 
The "default" device was still pointing to Pipewire. I nedeed to include a .asoundrc to indicate pulseaudio as default.
Ps.: this was needed only for Musescore 4, other programs (even Musescore 3.6) allow the selection of Jack or Alsa as an output (I think, a better solution).

`# Default to PulseAudio

pcm.!default {
    type pulse
    fallback "sysdefault"
    hint {
        show on
        description "Default ALSA Output (currently PulseAudio Sound Server)"
    }
}

ctl.!default {
    type pulse
    fallback "sysdefault"
}`
